.github/workflows/ci.yml

```diff
@@ -53,25 +94,32 @@ jobs:
           override: true
 
       - uses: Swatinem/rust-cache@v1
-
+      
       - name: Install cargo-nextest
         uses: taiki-e/install-action@nextest
 
       - name: Run cargo nextest
         uses: actions-rs/cargo@v1
         with:
           command: nextest
-          args: run
+          args: run --package nixpacks --test generate_plan_tests
+      - name: List docker tests (prep test-docker)
+        id: docker-prep
+        run: |
+          echo "::set-output name=matrix::{\"tests\":[\"$(cargo test -q --package nixpacks --test docker_run_tests -- --list --format=terse | sed -z 's/: test\n/\", \"/g' | sed 's/...$//')]}"
```

Comment:
For future selves, maybe a good idea adding a comment explaining wtf this is doing

Reply:
Done!